data.py

```diff
@@ -0,0 +1,66 @@
+import pymongo
+import pprint
+
+from pandas import DataFrame
+
+from dash.exceptions import PreventUpdate
+
+class MongoManager():
+    def __init__(self, server='localhost', port='27017'):
+        self._client = pymongo.MongoClient(server, int(port))
+
+    def _get_database_names(self, ):
```

Comment:
Public method names shouldn't start with an underscore, we can remove them from the beginning of all each method used from AppLogic.